tests/utils/index.js

```diff
@@ -273,37 +272,28 @@ const deleteAll = (except) => {
     });
 };
 
-const refreshToGetNewSettings = () => {
+const refreshToGetNewSettings = async () => {
   // wait for the updates to replicate
-  const dialog = element(by.css('#update-available .submit:not(.disabled)'));
-  return browser
-    .wait(protractor.ExpectedConditions.elementToBeClickable(dialog), 10000)
-    .then(() => dialog.click())
-    .catch(() => {
+  const dialog = $('#update-available .submit:not(.disabled)');
+  //await dialog.waitForClickable();
```

Comment:
I think this comment can be removed, right?

Reply:
Correct, thank you 🙂